### PR TITLE
Added unit tests for the Splash component

### PR DESCRIPTION
**Pull request by melissalupu**
Added a set of unit tests to ensure that the correct content appears in the Splash page. I have also included a test to check for the logic when the user clicks on the page:

image

### CODE REVIEW

File: src/components/splash/splash.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+});
+
+it('should display welcome text ', () => {
+  render(<Splash />);
+  
+  expect(screen.getByTestId('welcome')).toHaveTextContent('Welcome to the next generation of music creation');
+  expect(screen.getByText(/mélodie/i)).toBeInTheDocument();
+});
+
+it('should display the app logo', () => {
```
Comment (flynnn231) [inline src/components/splash/splash.test.js:21]: This is a great set of tests for the logo

**Review by flynnn231 — APPROVED** [review]
Good use of assertions for Jest and the React Testing Library. Please include more tests for other components.

**Review by elijaholan — APPROVED** [review]
Looks good Melissa, let's work towards adding more tests

**Review by Jsimpada — APPROVED** [review]
This contains a variety of assertions, great work :)

File: src/components/splash/splash.js [hunk 2]
```diff
@@ -10,18 +10,18 @@ function Splash(props) {
     };
 
     return (
-        <div id='splash' onClick={clearSplash} className={fade ? 'removed' : ''}>
+        <div id='splash' data-testid="screen" onClick={clearSplash} className={fade ? 'removed' : ''}>
```
Comment (brunogrcsada) [inline src/components/splash/splash.js:13]: We need to add test ids to the other refactored components!

**Review by brunogrcsada — APPROVED** [review]
All good with me, I'm happy for the pull request to be merged :+1:, we need to have a call to finalise the tests for the other components.

**Pull request merged**